From d03efbd04445db7e9853ef46a56b990ced9ed4cf Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@debian.org>
Date: Mon, 31 Oct 2011 19:21:27 +0000
Subject: Add GNU/Hurd hints for NDBM_File

currently, there are few tests failing because of a broken NDBM module;
using the same hints as used on Linux (given that apparently the cause
is the same) makes the following tests pass:
  t/op/dbm.t
  cpan/autodie/t/dbmopen.t
  cpan/Memoize/t/errors.t
  cpan/Memoize/t/tie_ndbm.t
  ext/NDBM_File/t/ndbm.t
  lib/AnyDBM_File.t

Dominic Hargreaves adds: Note that the original git commit log [what
you see above] lists some tests which aren't affected in blead.

Bug-Debian: http://bugs.debian.org/645989
Bug: https://rt.perl.org/rt3/Public/Bug/Display.html?id=102680
Origin: http://perl5.git.perl.org/perl.git/commit/f0d0a205cc02c769ca48d6df00f3eea304ff91d8

Patch-Name: fixes/hurd-ndbm.diff
---
 MANIFEST                   |    1 +
 ext/NDBM_File/hints/gnu.pl |    1 +
 2 files changed, 2 insertions(+)
 create mode 100644 ext/NDBM_File/hints/gnu.pl

diff --git a/MANIFEST b/MANIFEST
index ed6e37e..9ae00dc 100644
--- a/MANIFEST
+++ b/MANIFEST
@@ -3461,6 +3461,7 @@ ext/NDBM_File/hints/dec_osf.pl	Hint for NDBM_File for named architecture
 ext/NDBM_File/hints/dynixptx.pl	Hint for NDBM_File for named architecture
 ext/NDBM_File/hints/gnukfreebsd.pl	Hint for NDBM_File for named architecture
 ext/NDBM_File/hints/gnuknetbsd.pl	Hint for NDBM_File for named architecture
+ext/NDBM_File/hints/gnu.pl	Hint for NDBM_File for named architecture
 ext/NDBM_File/hints/linux.pl	Hint for NDBM_File for named architecture
 ext/NDBM_File/hints/sco.pl	Hint for NDBM_File for named architecture
 ext/NDBM_File/hints/solaris.pl	Hint for NDBM_File for named architecture
diff --git a/ext/NDBM_File/hints/gnu.pl b/ext/NDBM_File/hints/gnu.pl
new file mode 100644
index 0000000..db63567
--- /dev/null
+++ b/ext/NDBM_File/hints/gnu.pl
@@ -0,0 +1 @@
+do './hints/linux.pl' or die $@;
